contracts/upgradeable_contracts/BasicBridge.sol

```diff
@@ -1,104 +1,104 @@
 pragma solidity 0.4.24;
-import "../IBridgeValidators.sol";
-import "./OwnedUpgradeability.sol";
+import "../interfaces/IBridgeValidators.sol";
+import "./Upgradeable.sol";
 import "../upgradeability/EternalStorage.sol";
-import "../libraries/SafeMath.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./Validatable.sol";
 import "./Ownable.sol";
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol";
+import "./Claimable.sol";
 
-
-contract BasicBridge is EternalStorage, Validatable, Ownable, OwnedUpgradeability {
+contract BasicBridge is EternalStorage, Validatable, Ownable, Upgradeable, Claimable {
     using SafeMath for uint256;
 
     event GasPriceChanged(uint256 gasPrice);
     event RequiredBlockConfirmationChanged(uint256 requiredBlockConfirmations);
     event DailyLimitChanged(uint256 newLimit);
     event ExecutionDailyLimitChanged(uint256 newLimit);
 
-    function getBridgeInterfacesVersion() public pure returns(uint64 major, uint64 minor, uint64 patch) {
+    function getBridgeInterfacesVersion() external pure returns (uint64 major, uint64 minor, uint64 patch) {
```

Comment:
Should we update bridge contract version?

Reply:
Planned to ask you to do this as soon as we merge this PR